File: zad3.py
def sum_of_digits(num):
    # нахождение суммы цифр числа
    sum = 0
    while num != 0:
        sum += num % 10
        num //= 10
    return sum

# ввод чисел
n = 0
while n <= 0:
    n = int(input("Введите кол-во чисел(Больше 0): "))
    
nums = []
print("Введите числа:")
for i in range(n):
    nums.append(float(input()))
    
print("Числа:")
for num in nums:
    if sum_of_digits(num) < 10:
        print(num, end=" ")

File: zad3.swift
import Foundation

func sumOfDigits(_ num: Int) -> Int { // нахождение суммы цифр
    var sum = 0
    var number = num
    while number != 0 {
        sum += number % 10
        number /= 10
    }
    return sum
}

func main() {
    // ввод чисел
    var n = 0
    repeat {
        print("Введите кол-во чисел(Больше 0): ", terminator: "")
        if let input = readLine(), let inputInt = Int(input) {
            n = inputInt
        }
    } while n <= 0
    var nums = [Double]()
    print("Введите числа: ", terminator: "")
    for _ in 0..<n {
        if let input = readLine(), let inputDouble = Double(input) {
            nums.append(inputDouble)
        }
    }

    print("Числа: ", terminator: "")
    for num in nums { // перебор
        if sumOfDigits(Int(num)) < 10 { // проверка условия
            print("\(num) ", terminator: "") // вывод подходящих
        }
    }
}

main()